### ISSUE TITLE
Accessibility issues in Webdatarocks

### ISSUE THREAD

**Issue by playitsaiff** [WebDataRocks/pivot-react#34]
1.For all wdr tabs(#wdr-tab-format  ,#wdr-tab-options,  #wdr-tab-fields, #wdr-tab-fullscreen ) -> List item does not have a `<ul>`,` <ol>` parent element."

2.For all wdr tab link text Element has insufficient color contrast of 3.54 (foreground color: #888888, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1.

3.For all .wdr-sheet-heade Element has insufficient color contrast of 2.52 (foreground color: #999999, background color: #f1f1f1, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

**Comment by maksymkaD** [comment]
Hello!

Thank you for reporting these issues. 

1. We will add this issue to a backlog.

2,3. There is a way to set up a custom theme in WebDataRocks using our custom theme builder or manually edit the CSS file. Please refer to this [documentation page](https://www.webdatarocks.com/doc/custom-report-themes/), where both approaches are described.
Also, we recommend you check out the [accessibility features](https://www.flexmonster.com/doc/accessibility-overview/), such as high-contrast modes, in our other product - Flexmonster.

Please let us know if you have more questions.

Best Regards,
Maksym
